fixes/consolidate-multiple-description-history.sql

```diff
@@ -56,7 +56,7 @@ select _.ID, CurrentAuditID, NextAuditID
 from #suspect _
 join dbo.BaseAsset A on A.ID=_.ID and A.AuditBegin=_.CurrentAuditID
 join dbo.BaseAsset B on B.ID=_.ID and B.AuditBegin=_.NextAuditID
-join dbo.BaseAsset C on C.ID=_.ID and C.AuditEnd=_.CurrentAuditID AND ISNULL(C.[Description],-1) != ISNULL(A.[Description],-1)
+join dbo.BaseAsset C on C.ID=_.ID and C.AuditEnd=_.CurrentAuditID
```

Comment:
I'm confused.  What scenario are we targeting, exactly?

A Description-only change, followed quickly by a same-user Description-plus change?  If so, I don't think this commit does that.